chains/evm/cli/bridge/register-resource.go

```diff
@@ -22,62 +21,74 @@ var registerResourceCmd = &cobra.Command{
 		txFabric := evmtransaction.NewTransaction
 		return RegisterResourceCmd(cmd, args, txFabric)
 	},
+	Args: func(cmd *cobra.Command, args []string) error {
+		err := validateRegisterResourceFlags(cmd, args)
+		if err != nil {
+			return err
+		}
+
+		err = processRegisterResourceFlags(cmd, args)
+		if err != nil {
```

Comment:
There is no need for this check, you can just return `err` as it will be `nil` if everything is ok